base/shell/explorer/traywnd.cpp

```diff
@@ -2855,6 +2875,7 @@ class CTrayWindow :
         MESSAGE_HANDLER(WM_HOTKEY, OnHotkey)
         MESSAGE_HANDLER(WM_NCCALCSIZE, OnNcCalcSize)
         MESSAGE_HANDLER(TWM_SETTINGSCHANGED, OnTaskbarSettingsChanged)
+        MESSAGE_HANDLER(WM_SETTINGCHANGE, OnSettingsChange)
```

Comment:
Please move it to the line above, to group all the "WM_*" messages together.

Reply:
OK.